src/base-input.js

```diff
@@ -482,14 +493,29 @@ export default class BaseInput extends LitElement {
    * @returns {string} Error string.
    */
   getErrorMessage() {
-    if (this.error) {
-      return this.error;
-    }
+    let message = '';
+
     if (this.setCustomValidity) {
-      return this.setCustomValidity;
+      // return this.setCustomValidity;
+      message = this.setCustomValidity;
+    } else if (this.error) {
+      // return this.error;
+      message = this.error;
+    } else {
+      // return this.internalError;
+      message = this.internalError;
     }
 
-    return this.internalError;
+    this.dispatchEvent(new CustomEvent('auroInput__helpText', {
```

Comment:
This event should be named differently - this is not the helptext, but the error message.

Also, do we really need two separate events for this? Is there a way to combine into a single event? It might require some refactoring, but seems more intuitive.